### PR TITLE
Fix camera crash when mic permission is granted

### PR DESCRIPTION
**Pull request by ascendingSun**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Virtual device Pixel XL, Android 11.0
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### Description
Fixes signalapp/Signal-Android#10642
Currently the app crashes because mAudioRecorder is null when startRecording() is called in androidx\camera\core\VideoCapture.java.

E/SignalUncaughtExceptionHandler: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.AudioRecord.startRecording()' on a null object reference
        at androidx.camera.core.VideoCapture.startRecording(VideoCapture.java:368)
        at androidx.camera.view.SignalCameraXModule.startRecording(SignalCameraXModule.java:346)
        at androidx.camera.view.SignalCameraView.startRecording(SignalCameraView.java:481)

The fix is to rebind SignalCameraView onResume.

Tested on Android Emulator: Pixel XL (API 30)

### CODE REVIEW

File: app/src/main/java/org/thoughtcrime/securesms/mediasend/CameraXFragment.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
   @Override
   public void onResume() {
     super.onResume();
-
+    camera.bindToLifecycle(getViewLifecycleOwner());
```
Comment (jpventura) [inline app/src/main/java/org/thoughtcrime/securesms/mediasend/CameraXFragment.java:132]: Blank space added

**Review by jpventura — CHANGES_REQUESTED** [review]
- Nice work
  - Minor double-check at camera code.

**Comment by ascendingSun** [comment]
Thanks for taking a look.

I double checked about camera.setCameraLensFacing. _mCameraModule_._mCameraLensFacing_ value is retained when onResume is called. So I dont think it needs to be called again.

**Comment by alan-signal** [comment]
Merged in 24b5bac58924ec2326fec8a89163a1ecb0b49cfa

Thank you!

**Pull request closed without merging**